tests/integration/annotation_import/test_data_types.py

```diff
@@ -149,6 +164,67 @@ def test_import_data_types(client, configured_project,
     data_row.delete()
 
 
+@pytest.mark.parametrize('data_type_class',
+                         [AudioData, HTMLData, ImageData, TextData, VideoData])
+def test_import_data_types_v2(client, configured_project,
+                              data_row_json_by_data_type,
+                              annotations_by_data_type, data_type_class,
+                              v2_exports_by_data_type):
+
+    project_id = configured_project.uid
+
+    data_type_string = data_type_class.__name__[:-4].lower()
+    data_row_ndjson = data_row_json_by_data_type[data_type_string]
+    dataset = next(configured_project.datasets())
+    data_row = dataset.create_data_row(data_row_ndjson)
+
+    annotations_ndjson = annotations_by_data_type[data_type_string]
+    annotations_list = [
+        label.annotations
+        for label in NDJsonConverter.deserialize(annotations_ndjson)
+    ]
+    labels = [
+        lb_types.Label(data=data_type_class(uid=data_row.uid),
+                       annotations=annotations)
+        for annotations in annotations_list
+    ]
+
+    label_import = lb.LabelImport.create_from_objects(
+        client, project_id, f'test-import-{data_type_string}', labels)
+    label_import.wait_until_done()
+
+    assert label_import.state == AnnotationImportState.FINISHED
+    assert len(label_import.errors) == 0
+
+    num_retries = 5
+    task = None
+    while (num_retries > 0):
+        task = configured_project.export_v2(params={
+            "performance_details": False,
+            "label_details": True
+        })
+        task.wait_till_done()
```

Comment:
Do we have to rerun the task, or can we just keep checking the same task?

Reply:
Probably need to rerun the task, because the export runs can come up empty, if data isn't ready